README.adoc

```diff
@@ -10,77 +10,117 @@ image::https://github.com/rh-impact/pl-heudiconv/actions/workflows/ci.yml/badge.
 
 ## Abstract
 
-`pl-heudiconv` is a link:https://chrisproject.org/[_ChRIS^] _ds_ plugin that organizes brain imaging data into structured directory layouts.
+`pl-heudiconv` is a link:https://chrisproject.org/[_ChRIS_^] ds plugin that organizes brain imaging data into structured directory layouts.
+
+## Description
+The original `heudiconv` CLI tool is a flexible DICOM converter for organizing brain imaging data into structured directory layouts.
+This `pl-heudiconv` plugin is a wrapper that makes the `heudiconf` CLI tool available as a link:https://chrisproject.org/[_ChRIS_^] plugin.
+
+When running the `pl-heudiconv` plugin, you must specify two directories:
+
+* The _input directory_, where the brain imaging data is stored in DICOM files (with the `.dcm` file extension).
+* The _output directory_, where the structured directory layouts are generated.
+
+Both of these directories must already exist before you run the plugin.
+
+The documentation for the original `heudiconv` CLI tool is available from the following website:
+https://heudiconv.readthedocs.io/en/latest/index.html
+
+NOTE: In the original `heudiconv` CLI tool, the input and output directories are _named_ arguments, but link:https://chrisproject.org/[_ChRIS_^] plugins expect them to be _positional_ arguments. As a result of this translation, some other arguments might be missing, or structured or named differently.
+
+NOTE: The `pl-heudiconv` plugin does _not_ yet support the `--bids` argument, so you cannot currently use this plugin to convert the DICOM data to BIDS format.
```

Comment:
```suggestion
```

In my understanding, this is not correct. It supports it as much as the upstream heudiconv program supports it.

Reply:
Done, thanks :)